File: Bubble_sort.cpp
#include<iostream>
using namespace std;

void sort(int arr[],int size){

    for(int i=1;i<size;i++){

        bool swapped=false;

        for(int j=0;j<size-i;j++){

            if(arr[j]>arr[j+1])
            swap(arr[j],arr[j+1]);
            swapped=true;
        }
        if(swapped==false)
            break;
    }
}

void print_array(int arr[],int size){
    for(int i=0;i<size;i++){
        cout<<arr[i]<<" "<<endl;
    }
}

int main(){

    int list[5]={5,2,6,4,5};
    int size=sizeof(list)/sizeof(list[0]);
    sort(list,5);
    print_array(list,5);
}

File: Bubble_sort.py

def sort(arr,size):
   for i in range(1,size):
       for j in range(size-i):
           if arr[j]>arr[j+1]:
               arr[j],arr[j+1]=arr[j+1],arr[j]


unsorted_elements=[4,2,5,3,3,5,7,9,2,3,6,8,1]
size=len(unsorted_elements)

sort(unsorted_elements,size)

print(f"Sorted array are {unsorted_elements} ")